### PR TITLE
chore: simplify bug template

### PR DESCRIPTION
**Pull request by rarkins**
## Changes:

Simplifies existing bug template

## Context:

I noticed there was duplicate information requested

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]: The comment here can be removed, I agree!

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: I see you've removed the platform checkboxes (Azure, Bitbucket, etc.), what is the reason that you're removing those?

My reasoning to have the checkboxes is to make it easy to find the supported platforms, and the ways in which that platform can be used.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]: So we also need to know the Renovate version if people are using the hosted app?

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (rarkins) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Added back

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (rarkins) [reply .github/ISSUE_TEMPLATE/bug_report.md:32]: No

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]:
```suggestion
- [ ] GitLab self-hosted
```

A typo slipped in. 😉

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:67]:
```suggestion
<!-- Add any other context about the problem here, including your own debugging or ideas on what went wrong. -->
...
```

I think the comment and the `...` can be swapped around?

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Some small fixes.

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.md:68]:
```suggestion

...
```

Add newline to resolve the linter fail. 😄

**Review by HonkingGoose — APPROVED** [review]

**Comment by HonkingGoose** [comment]
@rarkins This won't auto-merge because its waiting for an approving review by a reviewer with **write** level access. My approving review is only at **triage** level access. 😄

**Pull request merged**
